corehq/apps/export/forms.py

```diff
@@ -970,47 +969,11 @@ def get_model_filter(self, mobile_user_and_group_slugs, can_access_all_locations
             self.cleaned_data['date_range'],
         )
 
-    def _get_mapped_user_types(self, selected_user_types):
-        user_types = []
-        if HQUserType.ACTIVE in selected_user_types:
-            user_types.append(BaseFilterExportDownloadForm._USER_MOBILE)
-        if HQUserType.ADMIN in selected_user_types:
-            user_types.append(BaseFilterExportDownloadForm._USER_ADMIN)
-        if HQUserType.UNKNOWN in selected_user_types:
-            user_types.append(BaseFilterExportDownloadForm._USER_UNKNOWN)
-        if HQUserType.DEMO_USER in selected_user_types:
-            user_types.append(BaseFilterExportDownloadForm._USER_DEMO)
-
-        return user_types
-
     def get_edit_url(self, export):
         from corehq.apps.export.views.edit import EditNewCustomFormExportView
         return reverse(EditNewCustomFormExportView.urlname,
                        args=(self.domain_object.name, export._id))
 
-    def get_es_user_types(self, filter_form_data):
```

Comment:
Note: This and above are same as the methods in `EmwfFilterFormExport` which will be called now